airbyte-integrations/connectors/source-stripe/source_stripe/schemas/invoices.json

```diff
@@ -66,7 +66,7 @@
       "type": ["null", "boolean"]
     },
     "discount": {
-      "type": ["null", "string"]
+      "type": ["null", "object"]
```

Comment:
If we declare the `type: object` we should include the structure with `properties`, otherwise, normalization errors are expected. If you don't know the nested structure, please add `additionalProperties: true`.

Reply:
Thanks. I added `"additionalProperties": true` to the schema